src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Patterns.cs

```diff
@@ -20,7 +20,7 @@ public override BoundNode VisitIsPatternExpression(BoundIsPatternExpression node
         // Input must be used no more than once in the result. If it is needed repeatedly store its value in a temp and use the temp.
         BoundExpression MakeIsPattern(BoundPattern loweredPattern, BoundExpression loweredInput)
         {
-            var syntax = _factory.Syntax = loweredPattern.Syntax;
+            _ = _factory.Syntax = loweredPattern.Syntax;
```

Comment:
having this discard seems unnecessary.

Reply:
potentially something to be tweaked in the code fix for this case.

Reply:
to clarify.  it looks like the above should just change to `_factory.Syntax = loweredPattern.Syntax`.

Reply:
I am going to leave this one as is for now as this is a very rare case.